Before: 29e8c6dc90c6518056afaef3803b756d5d1872288a8a6546befb35ffda476a46
After: b23a60f243c439d9316eab179162b22e7506fc2a61fbd3c8526f3edb7eb197b0
Patch: add005204f36054b425298163eed5aa8e6973c1fafff37ec802d22f8d20979e8
--- lib/corvid/%name%_plugin.rb.tt	2012-08-21 10:53:29.008883721 +0530
+++ lib/corvid/%name%_plugin.rb.tt	2012-08-21 10:53:35.168879381 +0530
@@ -27,9 +27,4 @@
   feature_manifest ({
   })
 
-  # A list of features to install automatically when the plugin itself is installed.
-  #
-  # @return [Array<String>] An array of feature names. Do not include the plugin prefix.
-  auto_install_features %w[]
-
 end
